src/utils/users.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import Client from 'client';
+import {UsersTypes} from 'constants';
+const HTTP_UNAUTHORIZED = 401;
+
+export function profilesToSet(state, action) {
```

Comment:
I don't imagine this'll be used outside of the reducers so I think it'd make sense to restrict its scope by moving it to `reducers/helpers.js`